packages/addons-v5/lib/notify.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
```

Comment:
could we instead just have `@heroku-cli/notifications` default to having sound disabled? Maybe have it catch any error unless some kind of flag is sent in as well?

Reply:
@jdxcode how do you feel about doing these changes to `@heroku-cli/notifications` as separate refactor PR?  I think there might be some broader opportunities to refactor where notifications are being used.

Reply:
👍